tests/ui/test_theme_views.py

```
@@ -0,0 +1,405 @@
+    from rero_mef.theme.views import _format_value_with_url
+
+    value, url = _format_value_with_url("url", "https://data.rero.ch/001/A123456")
+    assert "(RERO)A123456" in value
+    assert url is None
+
+
+def test_format_value_with_url_url_external():
+    """Url key with a plain HTTPS URL returns it as the external URL."""
+    from rero_mef.theme.views import _format_value_with_url
+
+    value, url = _format_value_with_url("url", "https://example.com")
```

Ruff (0.15.7)

[warning] Unpacked variable `value` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)